packages/backend/src/database/migrations/20240822143117_add_scheduler_selected_tabs_column.ts

```diff
@@ -0,0 +1,14 @@
+import { Knex } from 'knex';
+import { SchedulerTableName } from '../entities/scheduler';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable(SchedulerTableName, (table) => {
+        table.jsonb('selected_tabs');
```

Comment:
can we specifiy this is `nullable` ?

Reply:
also, it is probably better to use a `                    .specificType('selected_tabs', 'TEXT[]')` type instead of jsonb